qt5/build.sh

```diff
@@ -1,96 +1,93 @@
 #!/bin/bash
 
-BIN=$PREFIX/lib/qt5/bin
-QTCONF=$BIN/qt.conf
-
-chmod +x configure
+# change qt.conf name for qt4 coexistence
+patch -p1 <$RECIPE_DIR/qt5_conf.patch
 
 if [ `uname` == Linux ]; then
     MAKE_JOBS=$CPU_COUNT
 
-    ./configure -prefix $PREFIX \
-                -libdir $PREFIX/lib \
-                -bindir $PREFIX/lib/qt5/bin \
-                -headerdir $PREFIX/include/qt5 \
-                -archdatadir $PREFIX/lib/qt5 \
-                -datadir $PREFIX/share/qt5 \
-                -L $PREFIX/lib \
-                -I $PREFIX/include \
-                -release \
-                -opensource \
-                -confirm-license \
-                -shared \
-                -nomake examples \
-                -nomake tests \
-                -verbose \
-                -no-libudev \
-                -gtkstyle \
-                -qt-xcb \
-                -qt-pcre \
-                -qt-xkbcommon \
-                -xkb-config-root $PREFIX/lib \
-                -dbus
+    # see https://doc.qt.io/qt-5/linux-requirements.html
+    ./configure \
+        -verbose \
+        -prefix $PREFIX \
+        -bindir $PREFIX/lib/qt5/bin \
+        -headerdir $PREFIX/include/qt5 \
+        -libdir $PREFIX/lib \
+        -archdatadir $PREFIX/lib/qt5 \
+        -datadir $PREFIX/share/qt5 \
+        -L $PREFIX/lib \
+        -R $PREFIX/lib \
+        -I $PREFIX/include \
+        -opensource \
+        -confirm-license \
+        -release \
+        -shared \
+        -nomake examples \
+        -nomake tests \
+        -skip enginio \
+        -qt-pcre \
+        -system-xkbcommon-x11 \
+        -no-gtkstyle \
+        -dbus \
+        -no-gstreamer
```

Comment:
I'm pretty confident with the list of options we're passing to configure. There are things we really need, like `-gtkstyle` (for Ubuntu), and `-qt-xcb` (to not depend on the xcb version present in the system).

So please revert to the previous list of options.